module/VuFind/src/VuFind/ILS/Driver/Innovative.php

```diff
@@ -364,83 +364,82 @@ public function getMyProfile($userinfo)
     public function patronLogin($username, $password)
     {
         // TODO: if username is a barcode, test to make sure it fits proper format
-        $enabled = $this->config['PATRONAPI']['enabled'] ?? false;
-        if ($enabled && strtolower($enabled) !== 'false') {
-            // use patronAPI to authenticate customer
-            $url = $this->config['PATRONAPI']['url'];
-
-            // build patronapi pin test request
-            $result = $this->sendRequest(
-                $url . urlencode($username) . '/' . urlencode($password) .
-                '/pintest'
-            );
-
-            // search for successful response of "RETCOD=0"
-            if (stripos($result, 'RETCOD=0') == -1) {
-                // pin did not match, can look up specific error to return
-                // more useful info.
-                return null;
-            }
+        // Avoid situation where string value false would evaluate to true
+        $enabled = $this->config['PATRONAPI']['enabled'] ?? 'false';
+        if (!$enabled || $enabled === 'false') {
```

Comment:
Should we do this for a little more tolerance (and closer parity to the original version?)

```suggestion
        if (!$enabled || strtolower(trim($enabled)) === 'false') {
```